fiftyone/core/cli.py

```diff
@@ -1984,71 +1976,83 @@ class DatasetZooInfoCommand(Command):
 
         # Print information about a zoo dataset
         fiftyone zoo datasets info <name>
+
+        # Print information about a remote zoo dataset
+        fiftyone zoo datasets info https://github.com/<user>/<repo>
+        fiftyone zoo datasets info <url>
     """
 
     @staticmethod
     def setup(parser):
         parser.add_argument(
-            "name", metavar="NAME", help="the name of the dataset"
-        )
-        parser.add_argument(
-            "-b",
-            "--base-dir",
-            metavar="BASE_DIR",
-            help=(
-                "a custom base directory in which to search for downloaded "
-                "datasets"
-            ),
+            "name_or_url",
+            metavar="NAME_OR_URL",
+            help="the name or remote location of the dataset",
         )
 
     @staticmethod
     def execute(parser, args):
-        name = args.name
+        name_or_url = args.name_or_url
 
-        # Print dataset info
-        zoo_dataset = fozd.get_zoo_dataset(name)
-        print(
-            "***** Dataset description *****\n%s"
-            % textwrap.dedent("    " + zoo_dataset.__doc__)
-        )
+        zoo_dataset = fozd.get_zoo_dataset(name_or_url)
 
-        # Check if dataset is downloaded
-        base_dir = args.base_dir
-        downloaded_datasets = fozd.list_downloaded_zoo_datasets(
-            base_dir=base_dir
-        )
+        try:
+            dataset_dir = fozd.find_zoo_dataset(name_or_url)
+        except:
+            dataset_dir = None
 
-        if zoo_dataset.has_tags:
-            print("***** Tags *****")
-            print("%s\n" % ", ".join(zoo_dataset.tags))
+        if zoo_dataset.is_remote:
+            _print_dict_as_table(zoo_dataset.metadata)
+            print("")
+        else:
+            description = textwrap.dedent("    " + zoo_dataset.__doc__)
+            if description:
+                print("***** Dataset description *****\n%s" % description)
 
-        if zoo_dataset.has_splits:
-            print("***** Supported splits *****")
-            print("%s\n" % ", ".join(zoo_dataset.supported_splits))
+            if zoo_dataset.has_tags:
+                print("***** Tags *****")
+                print("%s\n" % ", ".join(zoo_dataset.tags))
+
+            if zoo_dataset.has_splits:
+                print("***** Supported splits *****")
+                print("%s\n" % ", ".join(zoo_dataset.supported_splits))
 
         print("***** Dataset location *****")
-        if name not in downloaded_datasets:
-            print("Dataset '%s' is not downloaded" % name)
-        else:
-            dataset_dir, info = downloaded_datasets[name]
+        if dataset_dir is not None:
             print(dataset_dir)
-            print("\n***** Dataset info *****")
-            print(info)
+        else:
+            print("Dataset '%s' is not downloaded" % name_or_url)
```

Comment:
If the url is very long, this statement might look a little bit odd. Maybe we can put a cap on it
```
print(f"Dataset '{name_or_url[:20]}' is not downloaded")
```

Reply:
Hmm, we don't have a precedent of truncating long strings in logs so I'd prefer consistency in this case 😄